tests/test_site_config.py

```diff
@@ -0,0 +1,51 @@
+from unittest.mock import MagicMock
+
+import pytest
+from ocs.site_config import get_control_client
+
+
+class TestGetControlClient:
+    """Test site_config.get_control_client().
+
+    """
+    def test_none_client_type(self):
+        """None should assume 'http' appropriately."""
+        mock_site = MagicMock()
+        get_control_client('test', site=mock_site)
```

Comment:
Right, I think I wrote this before setting `client_type='http'` in the arguments. But yeah, was just making sure a test hit the line that checked when `client_type` was `None`. Removed since it's redundant now.

As for `site.hub.data` below, yeah, I was just trying to make sure if anyone has any clients that specify type cases are handled correctly. Totally crashes if site.hub.data can't be checked, so I used the Mock.